packages/contracts/package.json

```diff
@@ -34,6 +34,7 @@
     "clean:yarn": "yarn clean",
     "clean:forge": "forge clean",
     "build": "forge build",
+    "build:smt": "tsx ./scripts/prebuild/smtchecker_prebuild.ts && forge build",
```

Comment:
You should use use `npm-run-all` in order to stay consistent with how we do it in this same package.json for the other scripts. e.g. `run-s build:smt build`

Also use kebab case for the file names. So `smtchecker-prebuild.ts` for example.